### PR TITLE
Feature/machine user authentication

### PR DESCRIPTION
**Pull request by bastihav**
Do you have any better suggestions for endpoint names?

### CODE REVIEW

**Review by david-buderus — COMMENTED** [review]
We could use `/login/machine` as an endpoint

File: authentication-management.yaml [hunk 1]
```diff
@@ -118,13 +118,40 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenericError'
+  /login2:
```
Comment (ldklenner) [inline authentication-management.yaml]:
```suggestion
  /login/machine:
```

File: authentication-management.yaml [hunk 2]
```diff
@@ -163,6 +190,49 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/GenericError'
+  /refresh2:
```
Comment (ldklenner) [inline authentication-management.yaml]:
```suggestion
  /refresh/machine:
```

**Review by dhelmert — CHANGES_REQUESTED** [review]
There is a new type of GenericError, called MultipleAuthorizationError, which needs to be added to the 400er errors to all authorized calls

**Review by dhelmert — APPROVED** [review]

**Pull request merged**